_posts/2020-08-15-Named-Graphs.md

```diff
@@ -543,11 +550,10 @@ large effect on the result sets returned.
 Named Graphs allow IrishGen to partition the various MSS sources into
 their own sub-graphs.  This allows the user to interrogate one MS
 tradition or another, which can be important to certain investigations
-such as when a user is searching broadly for some information; for
-instance, a name is encountered in another text and the user wishes to
-know if the name appears in any genealogical corpus and any context or
-connections that person may have in the medieval Irish genealogies
-which will help understand and read the context of the other text.
+such as when a user is searching for specific information which is
+known to reside in a particular MS.  For instance, a name is found in
+the same MS and the user wishes to check the genealogies quickly to
+see if that name is found within it.
```

Comment:
Apologies - I must have been looking at an out-of-date version, somehow. Can I suggest rephrasing "for specific information which is known to reside in a particular MS" to "for information in a particular MS version"? More users are likely to be wanting to check whether a certain piece of information is present than trying to find some information they somehow already know to be there.

Reply:
Or, to be more explicit "to verify whether a certain piece of information exists in a certain MS version"?